transa.sh

```diff
@@ -14,21 +14,31 @@ DEFAULT="itau"
 
 exit_instructions() {
     >&2 echo "General usage:"
-    >&2 echo "  ${0} [<amount>] [itau|brou] [-j|--json]"
+    >&2 echo "  ${0} [<amount>] [itau|brou|bcu] [-j|--json]"
     >&2 echo ""
     >&2 echo "Usage examples:"
     >&2 echo "  ${0}"
     >&2 echo "  ${0} <amount>"
     >&2 echo "  ${0} <amount> brou"
     >&2 echo "  ${0} <amount> itau"
+    >&2 echo "  ${0} <amount> bcu"
     >&2 echo "  ${0} itau"
     >&2 echo "  ${0} brou"
+    >&2 echo "  ${0} bcu"
     >&2 echo ""
     >&2 echo "If using amount with \$, make sure to escape it:"
     >&2 echo "  ${0} \\\$20000"
     >&2 echo "  ${0} \\\$20000 brou"
 }
 
+seven_days_ago() {
+    if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
This failed for me with the following error
```
./transa.sh: 41: OSTYPE: parameter not set
```
This is due to a combination of two things:
1. The script is suggested (see shebang) to be ran by a posix-compliant shell, and [OSTYPE is a bash-specific variable](https://www.gnu.org/software/bash/manual/bash.html#Major-Differences-From-The-Bourne-Shell).
    - In my case (Ubuntu) `/bin/sh` points to the [Dash shell](https://wiki.archlinux.org/title/Dash).
2. The script is configured to fail when encountering an unset variable (`-o nounset` at the top)

A couple of alternatives/workarounds to get OS information could be:
a. Using [uname](https://www.gnu.org/software/coreutils/manual/html_node/uname-invocation.html#uname-invocation)
b. Inspecting `/etc/os-release`

Reply:
Will use `uname -o` which gives the same information as `OSTYPE`. Thanks for the comment.